src/test/java/org/gridsuite/modification/server/modifications/GenerationDispatchTest.java

```diff
@@ -34,17 +50,64 @@ public class GenerationDispatchTest extends AbstractNetworkModificationTest {
     private static final String ABC_ID = "ABC";
     private static final String NEW_GROUP1_ID = "newGroup1";
     private static final String NEW_GROUP2_ID = "newGroup2";
+    private static final String GEN1_NOT_FOUND_ID = "notFoundGen1";
+    private static final String GEN2_NOT_FOUND_ID = "notFoundGen2";
+    private static final UUID FILTER_ID_1 = UUID.randomUUID();
+    private static final UUID FILTER_ID_2 = UUID.randomUUID();
+    private static final UUID FILTER_ID_3 = UUID.randomUUID();
+    public static final String PATH = "/v1/filters/export";
+
+    @Autowired
+    ApplicationContext context;
+
+    @SneakyThrows
+    @Before
+    public void specificSetUp() {
+        FilterService.setFilterServerBaseUri(wireMockServer.baseUrl());
+    }
+
+    private IdentifiableAttributes getIdentifiableAttributes(String id) {
+        return IdentifiableAttributes.builder()
+            .id(id)
+            .type(IdentifiableType.GENERATOR)
+            .build();
+    }
+
+    private FilterEquipments getFilterEquipments(UUID filterID, String filterName,
+                                                 List<IdentifiableAttributes> identifiableAttributes,
+                                                 List<String> notFoundEquipments) {
+        return FilterEquipments.builder()
+            .filterId(filterID)
+            .filterName(filterName)
+            .identifiableAttributes(identifiableAttributes)
+            .notFoundEquipments(notFoundEquipments)
+            .build();
+    }
+
+    private List<FilterEquipments> getTestFilters() {
+        IdentifiableAttributes gen1 = getIdentifiableAttributes(GTH2_ID);
+        IdentifiableAttributes gen2 = getIdentifiableAttributes(GROUP1_ID);
+        IdentifiableAttributes gen3 = getIdentifiableAttributes(ABC_ID);
+        IdentifiableAttributes gen4 = getIdentifiableAttributes(GH3_ID);
+        IdentifiableAttributes gen5 = getIdentifiableAttributes(GEN1_NOT_FOUND_ID);
+        IdentifiableAttributes gen6 = getIdentifiableAttributes(GEN2_NOT_FOUND_ID);
+
+        FilterEquipments filter1 = getFilterEquipments(FILTER_ID_1, "filter1", List.of(gen1, gen2), List.of());
+        FilterEquipments filter2 = getFilterEquipments(FILTER_ID_2, "filter2", List.of(gen3, gen4), List.of());
+        FilterEquipments filter3 = getFilterEquipments(FILTER_ID_3, "filter3", List.of(gen5, gen6), List.of(GEN1_NOT_FOUND_ID, GEN2_NOT_FOUND_ID));
```

Comment:
should we check reporter logging for those 2 not found gens ?

Reply:
Not, that's not really necessary